### ISSUE TITLE
How to change the .scss

### ISSUE THREAD

**Issue by tstrickland415** [zachsoft/Ionic-Material#186]
For some reason when I try to make changes to the scss files they never show up. I even tried completely deleting lines of code and its styling will still be there. For example, I've completely deleted lines 22-24 of ionic-material/src/scss/elements/item.scss yet the `border-radius` is still in effect. And this is just one of a few changes I've been trying to make. I even tried running `ionic setup sass` again and still the original ionic-material styles will show. Is there anything special I'm supposed to be doing to see my changes reflect?

**Comment by blinksmith** [comment]
![trydisablechace](https://cloud.githubusercontent.com/assets/409976/14331426/67fef3f2-fc6f-11e5-9deb-b78d29952760.png)

**Comment by tstrickland415** [comment]
Ah thanks,  I'll give this a try when I begin working again

**Comment by tstrickland415** [comment]
Nope, this solution didn't work. It sort of just freezes my developer tools panel. I currently have to just manually override everything in my styles.css. If there's a better solution out there, I'd greatly appreciate it. Thanks for trying though @blinksmith

**Comment by Anshdesire** [comment]
Do check if there is some error comes whenever you change scss file.
